docs/resources/aviatrix_aws_tgw_connect.md

```diff
@@ -0,0 +1,54 @@
+---
+subcategory: "TGW Orchestrator"
+layout: "aviatrix"
+page_title: "Aviatrix: aviatrix_aws_tgw_connect"
+description: |- Creates and manages Aviatrix AWS TGW Connect connections
```

Comment:
I remember we change line in the description in other .md files. Is it required?